components/card.tsx

```diff
@@ -42,18 +42,17 @@ const Card = ({
     // </div>
 
     <label
-      className={`${styles.card} ${
-        isSelected ? styles.active : styles.inactive
-      }`}
+      className={`${styles.card} ${isSelected ? styles.active : styles.inactive}`}
       onClick={onChange ? () => onChange() : null}
     >
       <input
         name="plan"
         type="radio"
         className={styles.radio}
         checked={isSelected}
+        onChange={() => { }}
```

Comment:
does this help with something?

Reply:
actually giving an onChange event to label does not make sense.
onChange handler should put to input element instead of the label.
if not, it will cause errors in the console window of the web browser.